docker-compose.yml

```diff
@@ -468,6 +468,54 @@ services:
     ports:
       - "3170"
 
+  contacts:
+    image: govuk/contacts:${CONTACTS_COMMITISH:-master}
+    build:
+      context: apps/contacts-admin
+    depends_on:
+      - publishing-api
+      - mysql
+      - rummager
+      - diet-error-handler
+      - whitehall
+    environment:
+      << : *govuk-app
+      SENTRY_CURRENT_ENV: contacts
+      VIRTUAL_HOST: contacts.dev.gov.uk
```

Comment:
I think other apps communicate with contacts they'd expect contacts-admin as that's what it's called on integration